Target.java

```diff
@@ -1,17 +1,62 @@
-/*
- * File: Target.java
- * Name: 
- * Section Leader: 
- * -----------------
- * This file is the starter file for the Target problem.
- */
-
-import acm.graphics.*;
-import acm.program.*;
-import java.awt.*;
-
-public class Target extends GraphicsProgram {	
-	public void run() {
-		/* You fill this in. */
-	}
-}
+/*
+ * File: Target.java
+ * Name: 
+ * Section Leader: 
+ * -----------------
+ * This file is the starter file for the Target problem.
+ */
+
+import acm.graphics.*;
+import acm.program.*;
+import java.awt.*;
+
+public class Target extends GraphicsProgram {
+	
+	/*runs the program*/
+	public void run() {
+		
+		/*conversion from inches to pixels*/
+		double inchToPixel = 72;
+		
+		/*set radii and center coordinates*/
+		double outerRad = 1 * inchToPixel;
+		
+		double middleRad = 0.65 * inchToPixel;
+		
+		double innerRad = 0.30 * inchToPixel;
+		
+		double centerX = getWidth()/2;
+		
+		double centerY = getHeight()/2;
+		
+		/*draw the circles, add them to canvas and set their respective colors*/
+		
+		/*outerCircle*/
+		GOval outerCircle = new GOval (centerX - outerRad, centerY - outerRad, 2 * outerRad, 2 * outerRad);
+		
+		outerCircle.setFilled(true);
+		
+		outerCircle.setColor(Color.RED);
+		
+		add(outerCircle);
+		
+		/*middleCircle*/
+		GOval middleCircle = new GOval (centerX - middleRad, centerY - middleRad, 2 * middleRad, 2 * middleRad);
+		
+		middleCircle.setFilled(true);
+		
+		middleCircle.setColor(Color.WHITE);
+		
+		add(middleCircle);
+		
+		/*innerCircle*/
+		GOval innerCircle = new GOval (centerX - innerRad, centerY - innerRad, 2 * innerRad, 2 * innerRad);
+		
+		innerCircle.setFilled(true);
+		
+		innerCircle.setColor(Color.RED);
+		
+		add(innerCircle);
+		

```

Comment:
The outerCircle, middlerCircle and innerCircle drawing should have been done in one method that way you would not be repeating very similar code 3 times. So for example you could have defined a method called drawCircle as follows:
```
private void drawCircle(double centerX, double centerY, double r, Color c) {
    GOval circle = new GOval (centerX - r, centerY - r, 2 * r, 2 * r);
    circle.setFilled(true);
    circle.setColor(c);
    add(circle);
}
```

And in the run method you would have called it as so:
```
drawCircle(centerX, centerY, outerRad, Color.RED)
drawCircle(centerX, centerY, middleRad, Color.WHITE)
drawCircle(centerX, centerY, innerRad, Color.RED)
```